lib/build-parent-toc.js

```diff
@@ -83,6 +83,7 @@ function processFile(path, file, mode) {
     return '';
   }
 
+  toc = toc.replace('**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*\n\n','')
```

Comment:
I think so? It was not getting filtered out properly when the TOCs were not present. We might be able to accomplish the same thing by passing a title arg to `transform`... 🤔

Reply:
Yeah I wonder if we can just omit a title all together so we don't have to then later strip it.

Reply:
Well, that would change all of the TOCs, wouldn't it? We currently do send it in when we call doctoc...

I suppose once we stop running original recipe `doctoc`, it won't matter. Once they're all removed, we could just remove line 87 here rather than pass in a `title` arg to `transform`